tests/ci/run_check.py

```diff
@@ -180,6 +180,15 @@ def main():
         )
         print("::notice ::Cannot run")
         sys.exit(1)
+    else:
+        post_commit_status(
+            commit,
+            SUCCESS,
+            "",
+            "ok",
+            PR_CHECK,
+            pr_info,
+        )
```

Comment:
```suggestion
 
    post_commit_status(
        commit,
        SUCCESS,
        "",
        "ok",
        PR_CHECK,
        pr_info,
    )
```

`else` is redundant after `sys.exit`